packages/connect/src/router-transport.spec.ts

```diff
@@ -224,4 +225,25 @@ describe("createRoutesTransport", function () {
       });
     });
   });
+  describe("supports self describing method type", function () {
```

Comment:
Great call to add a test 🙏 

We're not really testing a feature of `router-transport.ts`, but a feature of `router.ts`. Can you instead move the test to a new `router.spec.ts` file? You can simply inspect `ConnectRouter.handlers` to verify.